libhsail-rt/Makefile.am

```diff
@@ -60,7 +60,8 @@ AM_CPPFLAGS = -I$(srcdir)/rt -I$(srcdir)/include/internal
 AM_CFLAGS = \
 	-I $(srcdir)/../include \
 	-I $(srcdir)/../libgcc \
-	-I $(MULTIBUILDTOP)../../gcc/include $(PTH_CFLAGS)
+	-I $(MULTIBUILDTOP)../../gcc/include $(PTH_CFLAGS) \
+	-I $(srcdir)/../gcc/config/i386
```

Comment:
Is there a variable for the target's dir to use here instead so it would not be i386 specific (although ftz mode switching currently works only with i386).